### PR TITLE
Feature/#7,12

### PR DESCRIPTION
**Pull request by SLDem**
Create functions for sending email using RabbitMQ.

### CODE REVIEW

**Review by lurak — APPROVED** [review]

File: StockDashboardAPI/__init__.py [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
 app = Flask(__name__)
+
 app.config.from_object(DevelopmentConfig)
+app.config.update(dict(
+    MAIL_SERVER='smtp.googlemail.com',
+    MAIL_PORT=587,
+    MAIL_USE_TLS=1,
+    MAIL_USERNAME='stockdashboard581@gmail.com',
+    MAIL_PASSWORD='stockdashboard',
+))
```
Comment (lhalam) [inline StockDashboardAPI/__init__.py]: move to env

File: StockDashboardAPI/__init__.py [hunk 2]
```diff
@@ -1,5 +1,18 @@
 from flask import Flask
```
Comment (lhalam) [inline StockDashboardAPI/__init__.py]: move to workers

File: StockDashboardAPI/rabbit/rabbit_channel.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import pika
+from StockDashboardAPI import app
+
+
+def connect_queue():
+    rabbitmq = pika.BlockingConnection(pika.ConnectionParameters('localhost'))
```
Comment (lhalam) [inline StockDashboardAPI/rabbit/rabbit_channel.py]: `'localhost'` use env

File: StockDashboardAPI/rabbit/rabbit_worker.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+               ".header {background-color: lightblue;color: white;" \
+               ".link {text-decoration: none;color: #2C66E1;}</style>" \
+               "</head>" \
+               "<body><div class='header flex-center'><h1>Hello %s!</h1></div>" \
+               "<p class='flex-center' style='color: blue;'>" \
+               "You've been invited to view a Stock Dashboard at the following link by %s:</p>" \
+               "<p class='flex-center link'>%s</p></body></html>" % (recipient, sender, link)
+    msg = Message(html=template,
+                  subject='Hello {}'.format(recipient),
+                  sender='stockdashboard581@gmail.com',
```
Comment (lhalam) [inline StockDashboardAPI/rabbit/rabbit_worker.py]: use env

File: StockDashboardAPI/rabbit/rabbit_worker.py [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+                  recipients=[recipient]
+                  )
+
+    with app.app_context():
+        mail.send(msg)
+    channel.basic_ack(delivery_tag=method.delivery_tag)
+
+
+channel.basic_consume(queue='email_queue', on_message_callback=send_email_message)
+channel.start_consuming()
```
Comment (lhalam) [inline StockDashboardAPI/rabbit/rabbit_worker.py]: move to __name__ ==  main

**Review by lhalam — CHANGES_REQUESTED** [review]
move to folder worker/email_sender
use SMTP lib

**Review by daniiloleshchuk — APPROVED** [review]

**Pull request closed without merging**